src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -1111,6 +1111,19 @@ def setArchDims(self, p1, p2):
             if self.dim2.Distance:
                 self.dim2.on()
 
+    def getCursorSize(self):
```

Comment:
For new functions added, please use `snake_case`. Ideally all functions should be changed to this style, but it takes time to refactor all modules.